### PR TITLE
About page

### PR DESCRIPTION
**Pull request by donaldong**
- Add Nav Link to the about page
- Link the logo to the landing page
- Create overview, officers, and meetings components
- Fix the landing page test by wrapping it in a router component

If this design is acceptable, we need to collect some profile images from the officers and add them to the site.

Screenshot:
screen shot 2019-02-24 at 7 25 01 pm

### CODE REVIEW

**Comment by donaldong** [comment]
Update the styling to change the margin size of the overview section: 
screen shot 2019-02-25 at 1 14 13 pm

**Comment by FernandoCaudillo10** [comment]
@donaldong I see at the top that you want to merge to `error404` from `about_page`? Did you mean `master`?

**Comment by donaldong** [comment]
@FernandoCaudillo10 Thanks for looking at this carefully! This is intentional because this PR depends on #11 (it uses react router). Creating a PR which merges into the temporary branch `error404` can make the changes more readable to the reviewers -- it won't include changes which has been published in #11. It is a workaround to stack PRs on GitHub.

**Comment by FernandoCaudillo10** [comment]
I see! That is really cool and good to know!

**Comment by donaldong** [comment]
Rebase on #31 and fix lint errors.

**Comment by donaldong** [comment]
Use axios to make requests to fetch data from the backend. No UI or functional changes.

**Comment by donaldong** [comment]
Rebase on top of #32

**Comment by donaldong** [comment]
After this PR is merged, we will need to update the getting started doc:

Now there's one more step to do: run the `load_fixture.sh` script. It will become a problem for windows users. We will need to port the script to PowerShell if we want to support developing the site on Windows.

File: frontend/src/about/about_officers.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const Officer = ({
+  email,
+  name,
+  profileImg,
+  role,
+}) => (
+  <div className="officer">
+    <img
+      src={profileImg}
+      alt={`${name}`}
```
Comment (donaldong) [inline frontend/src/about/about_officers.js]:
```suggestion
      alt={name}
```

**Pull request merged**